aioesphomeapi/model.py

```diff
@@ -757,6 +756,28 @@ class AlarmControlPanelEntityState(EntityState):
     )
 
 
+# ==================== TEXT ====================
+class TextMode(APIIntEnum):
+    TEXT = 0
+    PASSWORD = 1
+
+
+@dataclass(frozen=True)
```

Comment:
We could use the _frozen_dataclass_decorator here so it uses less ram on newer python